public/js/p3/resources/maage.css

```
@@ -0,0 +1,1629 @@
+  display: inline-block;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: .5rem .75rem;
+  text-align: center;
+}
+
+#bv-brc-home-container div.mid-links:nth-child(1)>a:nth-child(5) {
+  color: #436D58;
+  text-decoration: underline;
+}
+
+#bv-brc-home-container div.mid-links:nth-child(1)>a:nth-child(5):nth-child(5):hover {}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)